src/client/app/actions/units.ts

```diff
@@ -35,3 +55,39 @@ export function fetchUnitsDetailsIfNeeded(): Thunk {
 		return Promise.resolve();
 	};
 }
+
+export function submitEditedUnits(): Thunk {
```

Comment:
No you cannot edit more than one unit at a time but the function submitEditedUnits() is needed to edit a unit. It would be best to change this function to edit a single unit.